src/styles/tokens.css

```
@@ -0,0 +1,73 @@
+  --color-surface-muted: #23283a;
+  --color-border: #2d3245;
+  --color-text: #e8ebf4;
+  --color-text-muted: #9aa2b8;
+  --color-accent: #6d8bff;
+  --color-accent-strong: #8ca0ff;
+  --color-accent-contrast: #0b0e17;
+  --color-success: #3fcf94;
+  --color-success-strong: #63dba8;
+  --color-danger: #ff7a90;
+  --color-focus-ring: #6d8bff;
+  color-scheme: dark;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)